client/components/App/index.js

```diff
@@ -1,4 +1,5 @@
 import React, { PropTypes } from 'react'
+import 'foundation-apps/dist/css/foundation-apps.css'
```

Comment:
On another project, we had issues with foundation's base styles interfering with things like react-toolbox.  Should we restrict the scope of foundation here as well?